package.json

```diff
@@ -85,15 +85,15 @@
     },
     {
       "path": "./packages/runtime/dist/browser/index.js",
-      "limit": "312B",
+      "limit": "200B",
```

Comment:
im confused here - we actually remove more code now, currently in master this is:

```
➜  compiled git:(master) yarn bundlesize
yarn run v1.22.4
$ yarn build && yarn build:dead-code-elimination && size-limit
$ yarn build:main && yarn build:browser
$ TYPEOF_WINDOW='typeof window' ttsc --build packages/tsconfig.json
$ TYPEOF_WINDOW='"object"' ttsc --build packages/tsconfig.browser.json
$ cd test/dead-code-elimination && ttsc
✔ Adding to empty webpack project

  ./packages/runtime/dist/browser/index.js
  Size limit: 105 B
  Size:       105 B with all dependencies, minified and gzipped

  ./packages/runtime/dist/browser/index.js
  Size limit: 312 B
  Size:       312 B with all dependencies, minified and gzipped

  ./test/dead-code-elimination/dist/index.js
  Size limit: 680 B
  Size:       550 B with all dependencies, minified and gzipped

✨  Done in 35.52s.
```

uhhh?

Reply:
@pgmanutd ok its getting `true` instead of `false` - ill investigate

Reply:
fixed in ts-transform-define

Reply:
ahhhh `Boolean('false')` and `Boolean('true')` both will be true always.